plugins/interactive/displayconsts/payload.go

```diff
@@ -0,0 +1,26 @@
+package displayconsts
+
+import "fmt"
+
+// An ErrorMessage payload is returned from the socketURL if an error occurred
+// when connecting.
+type ErrorMessage struct {
+	Code    string `json:"code"`
+	Message string `json:"message"`
+}
+
+// Error returns a string presentation of the ErrorMessage satifying the error
+// interface.
+func (e *ErrorMessage) Error() string {
+	return fmt.Sprintf("%sError: %s", e.Code, e.Message)
+}
```

Comment:
I think it's implicitly there...All the print methods knows how to convert error to string.